src/Message/DirectAuthorizeRequest.php

```diff
@@ -54,6 +54,8 @@ protected function getBaseAuthorizeData()
         $data['DeliveryPhone'] = $card->getShippingPhone();
         $data['CustomerEMail'] = $card->getEmail();
 
+        $data = array_merge($data, $this->getItemData());
```

Comment:
You are right about the formatting the unit tax amount it should be 2dp
Thank you so much